src/main/java/se/sundsvall/webmessagecollector/service/scheduler/MessageCacheService.java

```diff
@@ -38,15 +53,33 @@ public void fetchMessages(String familyId) {
 
 		final var bytes = openEClient.getMessages(familyId, fromTimestamp, "");
 		final var messages = toMessageEntities(bytes, familyId);
-		messageRepository.saveAll(messages);
 
+		messages.forEach(messageEntity -> {
```

Comment:
Is there risk that this transaction becomes too big? looping to fetch all attachments and storing every one in the same transaction seems like a risk for OOM.